src/btree/bt_sync.c

```diff
@@ -384,9 +385,13 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
                 break;
 
             /* Traverse through the internal page for obsolete child pages. */
-            if (is_las && WT_PAGE_IS_INTERNAL(walk->page)) {
-                WT_WITH_PAGE_INDEX(session, ret = __sync_ref_int_obsolete_cleanup(session, walk));
-                WT_ERR(ret);
+            if (is_las) {
```

Comment:
Adding the refs visited stat here doesn't provide a count of all refs that are visited, during the internal page processing, we will go through each child page of that internal page. So, I feel it is better to place the counter in the `__sync_ref_is_obsolete()` function would be good.

Reply:
Fixed.